.changeset/fuzzy-lobsters-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kubernetes-models/pipelines-as-code": major
```

Comment:
I prefer start the version number from `0.1.0`. `major` is only used when there's a breaking change.

```suggestion
"@kubernetes-models/pipelines-as-code": minor
```